registry/remote/credentials/store.go

```diff
@@ -51,6 +51,7 @@ type Store interface {
 // in the config file.
 type DynamicStore struct {
 	config             *config.Config
+	configPath         string
```

Comment:
`config` has already has the information of the `path`. To avoid duplicated info, `*config.Config` should have a method named
```go
func (cfg *Config) Path() string {
    return cfg.path
}
```
and then
```go
func (ds *DynamicStore) ConfigPath() string {
	return ds.config.Path()
}
```

Reply:
Changed accordingly.